source/scss/layout/_main.scss

```diff
@@ -24,6 +24,16 @@ main {
     position: relative;
   }
 
+  .width-100 {
+    width: 100%;
+  }
+}
+
+@include media-breakpoint-up(sm) {
```

Comment:
Please write it like so:
```
.width-25 {
  @include media-breakpoint-up("sm") {
    flex: 0 0 50%;
    max-width: 50%;
  }
}
```

media break point inside the class. We want to achieve a better code styling. :)

Reply:
fixed while adding a class for standard-content